src/main/java/baseball/GameManager.java

```diff
@@ -0,0 +1,71 @@
+package baseball;
+
+import baseball.generator.RandomNumberGenerator;
+import baseball.receiver.ConsoleInputReceiver;
+import baseball.receiver.InputReceiver;
+import baseball.number.GameNumber;
+import baseball.printer.ConsolePrinter;
+import baseball.printer.Printer;
+import baseball.result.Result;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class GameManager {
+    private final Printer printer;
+    private final InputReceiver receiver;
+    private final GameNumber seedNumber;
+    private final GameNumber userNumber;
+
+    public GameManager(Scanner scanner) {
+        if(scanner == null)
+            throw new NullPointerException();
```

Comment:
컨벤션을 지켜주세요 🙂

[중괄호](https://naver.github.io/hackday-conventions-java/#braces-knr-style)
[공백 삽입](https://naver.github.io/hackday-conventions-java/#space-around-brace)

```suggestion
        if (scanner == null) {
            throw new NullPointerException();
        }
```